### DISCUSSION TITLE
Feature Request: Add 'showClear' functionality to MultiSelect

### DISCUSSION THREAD

**Discussion by zach-mcnulty** [primefaces/community#689] [PrimeVue]
Since the MultiSelect component looks a lot like the Dropdown component, it would be nice to have the ability to add :showClear="true" to the MultiSelect to enable the ability to deselect all currently selected options? Thoughts, anyone?

<img width="1098" alt="image" src="https://github.com/primefaces/community/assets/72584137/fa6792f6-df70-4550-9c9d-339e05916a91">

**Comment by tugcekucukoglu** [comment]
We can add it to our roadmap.

Thank you.

**Comment by munkeywolf00** [comment]
I also request this feature. In general, any input field that has showClear or is clearable (if it makes sense) would be helpful.

**Comment by dblinov** [comment]
I'm also really need this functionality! Please add it!
